sdk/storage/azure-storage-file-share/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "java",
   "TagPrefix": "java/storage/azure-storage-file-share",
-  "Tag": "java/storage/azure-storage-file-share_dfe3610108"
+  "Tag": "java/storage/azure-storage-file-share_a570c2daf7"
```

Comment:
nit: In theory we should be able to migrate without having to change the test recordings, let's revert this an get the migration done without updating this (if possible). The theory behind sync-over-sync stack is that REST API calling patterns are functionally equivalent between synchronous and asynchronous SDK APIs, it's their threading model that changes.